pkg/pgmodel/end_to_end_tests/drop_test.go

```diff
@@ -5,6 +5,7 @@ package end_to_end_tests
 
```

Comment:
We should also add a test where the series is really truly deleted and the ingestor's cache handling correctly handles that and re-creates a new series

Reply:
👍 
I also want to add a test that the cache-trashing code works correctly...

Reply:
> We should also add a test where the series is really truly deleted and the ingestor's cache handling correctly handles that and re-creates a new series

added to the end of `TestSQLDropMetricChunk()`

Reply:
> I also want to add a test that the cache-trashing code works correctly...

added in `sql_tests.go`